src/link/sdas_obj.cpp

```diff
@@ -136,7 +136,8 @@ enum RelocFlags {
 };
 
 void sdobj_ReadFile(FileStackNode const &where, FILE *file, std::vector<Symbol> &fileSymbols) {
-	std::vector<char> line(256);
+	std::vector<char> line;
```

Comment:
Note to self, some other variables also have this problem:

```
src/gfx/process.cpp:    std::vector<Palette> palettes(nbPalettes);
src/gfx/process.cpp:    std::vector<Palette> palettes(options.palSpec.size());
src/link/object.cpp:    std::vector<std::unique_ptr<Section>> fileSections(nbSections);
test/gfx/randtilegen.cpp:       std::vector<uint8_t> data(height * 8 * width * 8 * SIZEOF_PIXEL);
test/gfx/randtilegen.cpp:       std::vector<uint8_t *> rowPtrs(height * 8);
```

Reply:
Is it a problem? Several of those variables *do* get filled up to capacity afterwards.

Reply:
I'm at least noting them here for me to double-check later.

Reply:
Okay, yeah, they all look fine.